onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2082,6 +2082,14 @@ ORT_API_STATUS_IMPL(OrtApis::CreateSessionFromArrayWithPrepackedWeightsContainer
   API_IMPL_END
 }
 
+ORT_API_STATUS_IMPL(OrtApis::GetTensorMemoryInfo, _In_ const OrtValue* value, _Out_ const OrtMemoryInfo** out) {
+  TENSOR_READ_API_BEGIN
+  const OrtMemoryInfo mem_info = tensor.Location();
```

Comment:
This will make a copy of the OrtMemoryInfo and return a reference to a local here?

Reply:
Good catch. Yeah, this should just be doing:

    *out=tensor.Location();

Reply:
Sure I have made the change